http/org.wso2.carbon.transport.http.netty/src/test/java/org/wso2/carbon/transport/http/netty/websocket/HttpToWebSocketProtocolSwitchTestCase.java

```diff
@@ -0,0 +1,98 @@
+/*
```

Comment:
HttpToWebSocketProtocolSwitchTestCase.java to HttpToWsProtocolSwitchTestCase.java